assets/app/views/builds.html

```diff
@@ -98,8 +114,36 @@
                       </dd>
                     </dl>
                   </span>
-                  <span ng-switch-when="imageChange">
-                    Image change on image repository {{trigger.imageChange.from.name}}:{{trigger.imageChange.tag}}
+                  <span ng-switch-when="imageChange">                    
+                    <div ng-switch="buildConfig.parameters.strategy.type">
+                      <div ng-switch-when="STI">
+                        <dl class="dl-horizontal" ng-if="buildConfig.parameters.strategy.stiStrategy.from&& buildConfig.parameters.strategy.stiStrategy.from.kind!='ImageStreamImage'">
+                          <dt>
+                            On builder update
+                          </dt>
+                          <dd> </dd>
+                          <br>
```

Comment:
we typically don't use \<br\> tags, why were they needed?  I'm guessing something to do with the empty \<dd\> tag?  @sg00dwin has a fix included in https://github.com/openshift/origin/pull/1756 that fixes the css around empty \<dd\> tags